### PR TITLE
Add djlsp vscode extension

### PR DESCRIPTION
**Pull request by rubenhesselink**

### CODE REVIEW

File: vscode/client/src/extension.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	].map(expandHomeDir);
+
+	let djlspPath = djlspPaths.find(fs.existsSync);
+
+	// Add error handling for when djlsp is not installed
+	if (!djlspPath) {
+		throw new Error('djlsp is not installed');
+	}
+
+	const djlspArgs = ['--enable-log'];
```
Comment (krukas) [inline vscode/client/src/extension.ts:39]: Make this an option, or remove it.

File: vscode/client/src/extension.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+			// Notify the server about file changes to '.clientrc files contained in the workspace
+			fileEvents: workspace.createFileSystemWatcher('**/.clientrc'),
+		},
+		initializationOptions: initializationOptions,
+	};
+
+	// Create the language client and start the client.
+	client = new LanguageClient(
+		'languageServerExample',
+		'Language Server Example',
```
Comment (krukas) [inline vscode/client/src/extension.ts:107]: Update with djlsp names :smile:

File: vscode/client/src/extension.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+	// This is also updated when the workspace folder is changed.
+	const workspaceFolders = workspace.workspaceFolders;
+	if (workspaceFolders === undefined) {
+		throw new Error('No workspace folder is open');
+	}
+	const workspaceFolder = workspaceFolders[0];
+	const workspaceFolderPath = workspaceFolder.uri.path;
+
+	// If the extension is launched in debug mode then the debug server options are used
+	// Otherwise the run options are used
```
Comment (krukas) [inline vscode/client/src/extension.ts:59]: Is this comment still correct? it looks like it uses the enableLogging option

Comment (rubenhesselink) [reply vscode/client/src/extension.ts]: No this comment is not correct anymore, it has now been removed.

**Pull request merged**